srcalc/src/main/java/gov/va/med/srcalc/vista/VistaPatientDao.java

```diff
@@ -28,7 +32,7 @@
      * characters so that the note is easily visible in CPRS without horizontal scrolling.
      * @param patient
      * @param noteBody 
-     * @return
+     * @return one of the above-defined return code strings
```

Comment:
Would you want to add links to the variables, or leave it out of the comment because the number of strings could grow?